test/unit/bitmap_test.c

```diff
@@ -0,0 +1,80 @@
+#include <runtime.h>
+#include <stdlib.h>
+#define EXIT_FAILURE 1
+#define EXIT_SUCCESS 0
+
+#define WHOLE_RANGE irange(0, infinity)
+
+
+/**
+ *  Tests allocation of bitmap using allocate_bitmap 
+ *  function.
+ */
+bitmap test_alloc() {
+    heap h = init_process_runtime();
+    bitmap b = allocate_bitmap(h, h, infinity);
+    if (b == INVALID_ADDRESS) 
+        msg_err("!!! allocation failed for bitmap\n");
+    return b;
+}
+
+/**
+ *  Tests cloning of bitmap using bitmap_clone 
+ *  function.
+ */  
+boolean test_clone(bitmap b) {
+    bitmap b_cpy = bitmap_clone(b);
+    if (b_cpy == INVALID_ADDRESS) {
+        msg_err("!!! cloning failed for bitmap\n");
+        return false;
+    }
+    return true;
+}
+
+/**
+ *  Tests copying of bitmap using bitmap_copy
+ *  function.
+ */ 
+boolean test_copy(bitmap b) {
+    heap h = init_process_runtime();
```

Comment:
`init_process_runtime()` is supposed to be called only once at initialization, you are calling it both in test_alloc() and here. I suggest calling it outside these functions and passing the returned heap as argument to these functions.